### ISSUE TITLE
npm install broken on master

### ISSUE THREAD

**Issue by ivanov** [jupyter/jupyterlab#1066]
```

12:25@jlab(master)$ npm install

> jupyterlab@0.5.0 postinstall /Users/pivanov5/code/jpy/jupyterlab
> node scripts/dedupe.js


> jupyterlab@0.5.0 prepublish /Users/pivanov5/code/jpy/jupyterlab
> npm run build


> jupyterlab@0.5.0 build /Users/pivanov5/code/jpy/jupyterlab
> npm run build:src


> jupyterlab@0.5.0 build:src /Users/pivanov5/code/jpy/jupyterlab
> tsc --project src && node scripts/copyfiles.js

node_modules/@types/requirejs/index.d.ts(416,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'require' must be of type 'NodeRequire', but here has type 'Require'.

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.6.0/bin/node" "/usr/local/bin/npm" "run" "build:src"
npm ERR! node v6.6.0
npm ERR! npm  v3.10.7
npm ERR! code ELIFECYCLE
npm ERR! jupyterlab@0.5.0 build:src: `tsc --project src && node scripts/copyfiles.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyterlab@0.5.0 build:src script 'tsc --project src && node scripts/copyfiles.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyterlab package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     tsc --project src && node scripts/copyfiles.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyterlab
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyterlab
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pivanov5/code/jpy/jupyterlab/npm-debug.log

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.6.0/bin/node" "/usr/local/bin/npm" "run" "build"
npm ERR! node v6.6.0
npm ERR! npm  v3.10.7
npm ERR! code ELIFECYCLE
npm ERR! jupyterlab@0.5.0 build: `npm run build:src`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyterlab@0.5.0 build script 'npm run build:src'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyterlab package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run build:src
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyterlab
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyterlab
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pivanov5/code/jpy/jupyterlab/npm-debug.log

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.6.0/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.6.0
npm ERR! npm  v3.10.7
npm ERR! code ELIFECYCLE
npm ERR! jupyterlab@0.5.0 prepublish: `npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyterlab@0.5.0 prepublish script 'npm run build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyterlab package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyterlab
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyterlab
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pivanov5/code/jpy/jupyterlab/npm-debug.log
```

**Comment by ivanov** [comment]
cc @blink1073 who worked on this in #1030

**Comment by blink1073** [comment]
Please try an `rm -rf node_modules/@types/requirejs` and try again.

**Comment by ivanov** [comment]
ok, I seem to have resolved this by fixing `clean:slate` target in #1067 which nuked all of node_modules.
